Classes/Steno.sc

```diff
@@ -604,14 +612,22 @@ Steno {
 
 	newSynth { |token, i, args|
 		var target = synthList[i - 1];
+		var addAction = if(target.isNil) {
+			 target = group;
+			\addToHead
+		} {
+			\addAfter
+		};
+
+
 		token = token.asSymbol;
 		if(encyclopedia.at(token).isNil) { token = '?' }; // silent
 
 		^Synth(
 			this.prefix(token),
 			args,
 			target: target, // use previous synth in list. if nil, this is default group.
```

Comment:
this comment isn't true anymore. maybe `// use previous synth in list. if nil, use group`.